WooCommerce/Classes/ViewRelated/Dashboard/Free Trial/StorePlanBannerPresenter.swift

```diff
@@ -24,7 +24,7 @@ final class FreeTrialBannerPresenter {
 
     /// Holds a reference to the Free Trial Banner view, Needed to be able to remove it when required.
     ///
-    private var freeTrialBanner: UIView?
+    private var storePlanBanner: UIView?
```

Comment:
non-blocking suggestion: If possible, it would be nice to add unit tests to validate the visibility of this banner based on this site and plan details.

Reply:
Unit testing requires significant effort to update `StorePlanSynchronizer` to make it mockable. This can be handled in a separate PR when we have more time.

Reply:
Sure. I understand.